dask/dataframe/tests/test_io.py

```diff
@@ -909,19 +901,27 @@ def test_to_csv_gzip():
             tm.assert_frame_equal(result, df)
 
 
-def test_to_csv_series():
-    s = pd.Series([1, 2, 3], index=[10, 20, 30], name='foo')
-    a = dd.from_pandas(s, 2)
-    with tmpfile('csv') as fn:
-        with tmpfile('csv') as fn2:
-            a.to_csv(fn)
-            s.to_csv(fn2)
-            with open(fn) as f:
-                adata = f.read()
-            with open(fn2) as f:
-                sdata = f.read()
+def test_to_csv_simple():
+    df0 = pd.DataFrame({'x': ['a', 'b', 'c', 'd'],
+                       'y': [1, 2, 3, 4]}, index=[1., 2., 3., 4.])
+    df = dd.from_pandas(df0, npartitions=2)
+    with tmpdir() as dir:
+        dir = str(dir)
+        df.to_csv(dir)
+        assert os.listdir(dir)
+        result = dd.read_csv(dir+'/*').compute()
+    assert (result.x == df0.x).all()
 
-            assert adata == sdata
+
+def test_to_csv_series():
```

Comment:
We don't yet have a consistent way of doing this across the `to_foo` functions.  You could pass get down if you felt like it, we could also just rely on globals/context managers.  The selection of `get_sync` used to be very valuable when we wrote to a single file by default.  Parallel writes to a sequential file often caused scheduling traffic jams.  This is no longer much of an issue, so passing down `get` isn't a big deal.

If it was me I'd just depend on globals.